modules/tooltip/main/index.coffee

```diff
@@ -0,0 +1,21 @@
+
+hx.tooltip = (selector, opts) ->
+
+  options = hx.merge({
+    message: ''
+    align: 'right'
+  }, opts)
+
+  content = (element) ->
+    hx.select(element)
+      .text(options.message)
+      .classed('hx-tooltip', true)
+
+  dropdownOptions = {
+    mode: 'hover',
+    align: options.align
+  }
+
+  new hx.Dropdown(hx.select(selector).node(), content, dropdownOptions)
```

Comment:
The string option for dropdown is interpreted as html, which something we should really deprecate and avoid in the future because of the security implications. Allowing the user to pass in a text-only string or a renderer function would make sense - but that would still need a custom content renderer function here until dropdown is updated - which can't happen until 2.0.0

I see what you mean about there not being much difference between dropdown and this. Could this be because dropdown just happens to have everything that tooltip needs to work _at the moment_. Extending dropdown would imply that this will always work using a dropdown, which might not always be the case (although probably will be :-p). 

I think it's better to keep things as separate modules as much as possible - even if it does use another module under the hood, the user doesn't need to know that - this gives us more flexibility to change how it works without affecting the api and the preconceptions the user has